2023-06-29 22:11:06,476 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-06-29 22:11:07,136 [INFO    ] Authentication (publickey) failed.
2023-06-29 22:11:07,379 [INFO    ] Connected (version 2.0, client OpenSSH_7.4p1)
2023-06-29 22:11:08,030 [INFO    ] Authentication (password) successful!
2023-06-29 22:11:08,052 [INFO    ] connected to host 92.53.64.66 base app_nedvizhka_db
2023-06-29 22:12:54,102 [INFO    ] обработка district_id:
2023-06-29 22:12:54,113 [INFO    ] districts_update:   0%|          | 0/117016 [00:00<?, ?it/s]
2023-06-29 22:13:02,645 [INFO    ] добавлен новый район "Углевка" в districts функцией district_from_rn_mkrn
2023-06-29 22:13:05,812 [INFO    ] добавлен новый район "Нефтяников" в districts функцией district_from_rn_mkrn
2023-06-29 22:13:09,113 [INFO    ] districts_update:  10%|9         | 11375/117016 [00:15<02:19, 758.32it/s]
2023-06-29 22:13:24,114 [INFO    ] districts_update:  26%|##5       | 30080/117016 [00:30<01:23, 1045.74it/s]
2023-06-29 22:13:34,156 [INFO    ] districts_update:  36%|###5      | 42041/117016 [00:40<01:11, 1045.74it/s]
2023-06-29 22:13:39,115 [INFO    ] districts_update:  41%|####      | 47879/117016 [00:45<01:02, 1110.04it/s]
2023-06-29 22:13:54,115 [INFO    ] districts_update:  57%|#####6    | 66155/117016 [01:00<00:44, 1152.81it/s]
2023-06-29 22:14:04,174 [INFO    ] districts_update:  67%|######7   | 78896/117016 [01:10<00:33, 1152.81it/s]
2023-06-29 22:14:09,115 [INFO    ] districts_update:  73%|#######2  | 85017/117016 [01:15<00:26, 1190.54it/s]
2023-06-29 22:14:24,216 [INFO    ] districts_update:  89%|########8 | 103809/117016 [01:30<00:11, 1190.54it/s]
2023-06-29 22:14:24,217 [INFO    ] districts_update:  89%|########8 | 103810/117016 [01:30<00:10, 1208.97it/s]
2023-06-29 22:14:34,234 [INFO    ] districts_update:  99%|#########9| 116277/117016 [01:40<00:00, 1208.97it/s]
2023-06-29 22:14:34,880 [INFO    ] districts_update: 100%|##########| 117016/117016 [01:40<00:00, 1161.25it/s]
2023-06-29 22:14:34,881 [INFO    ] district add from realty
2023-06-29 22:14:36,559 [INFO    ] Успешно обработан файл циан 28-06-23 (без дублей).csv для realty
2023-06-29 22:15:11,969 [INFO    ] Успешно обработан файл 28-06-23 (без дублей).csv для realty
2023-06-29 22:15:48,303 [INFO    ] попытка загрузить данные dadata из C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/cian_dadata_request_28_06_2023.csv
2023-06-29 22:15:48,303 [ERROR   ] нет исторических данных C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/cian_dadata_request_28_06_2023.csv - будет создан новый df для запросов к dadata
2023-06-29 22:16:01,321 [INFO    ] количество запросов к dadata составит: 4130
2023-06-29 22:16:01,327 [INFO    ] 0%|          | 0/4130 [00:00<?, ?it/s]
2023-06-29 22:16:01,616 [INFO    ] 100%|##########| 4130/4130 [00:00<00:00, 14291.61it/s]
2023-06-29 22:16:01,616 [INFO    ] dadata con closed succesfully
2023-06-29 22:16:01,624 [INFO    ] обращение к DDT по 3520/4130 адресам из cian заняло 0:00:13.345620
2023-06-29 22:16:05,270 [INFO    ] попытка загрузить данные dadata из C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_01_06_2023.csv
2023-06-29 22:16:05,270 [ERROR   ] нет исторических данных C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_01_06_2023.csv - будет создан новый df для запросов к dadata
2023-06-29 22:16:18,288 [INFO    ] количество запросов к dadata составит: 2545
2023-06-29 22:16:18,291 [INFO    ] 0%|          | 0/2545 [00:00<?, ?it/s]
2023-06-29 22:16:18,421 [ERROR   ] Traceback (most recent call last):
  File "C:\Users\pnknv\AppData\Local\Temp\ipykernel_2316\717108773.py", line 70, in dadata_request_dev
    wr.writelines(f"{row.ad_id}: {row.addr}" + ',\n')
  File "C:\Program Files\Python310\lib\encodings\cp1251.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u200e' in position 0: character maps to <undefined>
, try reconnect
2023-06-29 22:16:18,424 [INFO    ] 87%|########7 | 2222/2545 [00:00<00:00, 16795.42it/s]
2023-06-29 22:16:41,003 [INFO    ] попытка загрузить данные dadata из C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_28_06_2023.csv
2023-06-29 22:16:41,003 [ERROR   ] нет исторических данных C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_28_06_2023.csv - будет создан новый df для запросов к dadata
2023-06-29 22:16:54,023 [INFO    ] количество запросов к dadata составит: 2545
2023-06-29 22:16:54,026 [INFO    ] 0%|          | 0/2545 [00:00<?, ?it/s]
2023-06-29 22:16:54,145 [ERROR   ] Traceback (most recent call last):
  File "C:\Users\pnknv\AppData\Local\Temp\ipykernel_2316\717108773.py", line 70, in dadata_request_dev
    wr.writelines(f"{row.ad_id}: {row.addr}" + ',\n')
  File "C:\Program Files\Python310\lib\encodings\cp1251.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u200e' in position 0: character maps to <undefined>
, try reconnect
2023-06-29 22:16:54,146 [INFO    ] 87%|########7 | 2222/2545 [00:00<00:00, 18474.36it/s]
2023-06-29 22:19:35,649 [INFO    ] попытка загрузить данные dadata из C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_28_06_2023.csv
2023-06-29 22:19:35,650 [ERROR   ] нет исторических данных C:/Users/pnknv/Documents/GitHub/yadisk_loader/dev_check_jupyter/saved_data_csv/avito_dadata_request_28_06_2023.csv - будет создан новый df для запросов к dadata
2023-06-29 22:19:48,670 [INFO    ] количество запросов к dadata составит: 2545
2023-06-29 22:19:48,673 [INFO    ] 0%|          | 0/2545 [00:00<?, ?it/s]
2023-06-29 22:19:48,805 [INFO    ] 100%|##########| 2545/2545 [00:00<00:00, 19233.86it/s]
2023-06-29 22:19:48,806 [INFO    ] dadata con closed succesfully
2023-06-29 22:19:48,810 [INFO    ] обращение к DDT по 2389/2545 адресам из avito заняло 0:00:13.179333
